### PR TITLE
Add support for PGP

### PR DESCRIPTION
**Pull request by akime-**
PGP description https://en.wikipedia.org/wiki/Pretty_Good_Privacy
mime description https://tools.ietf.org/html/rfc3156

FIxes https://github.com/sindresorhus/file-type/issues/336

### CODE REVIEW

File: core.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /// <reference types="node"/>
-import {Readable as ReadableStream} from 'stream';
-import {ITokenizer} from 'strtok3/lib/core';
+import { Readable as ReadableStream } from 'stream';
+import { ITokenizer } from 'strtok3/lib/core';
```
Comment (sindresorhus) [inline core.d.ts:3]: Don’t do unrelated changes

Comment (akime-) [reply core.d.ts]: Spaces removed

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/file-type/pull/367/files#diff-1dd241c4cd3fd1dd89c570cee98b79ddR16-R17

File: package.json [hunk 2]
```diff
@@ -204,4 +205,4 @@
 			"prefer-named-capture-group": "off"
 		}
 	}
-}
+}
```
Comment (sindresorhus) [inline package.json:208]: Unrelated change

Comment (akime-) [reply package.json]: Fixed

**Comment by sindresorhus** [comment]
I commented https://github.com/sindresorhus/file-type/pull/367/files#issuecomment-636673451 for a reason. Please read it.
